### ISSUE TITLE
Use FORECA weather API ⛅

### ISSUE THREAD

**Issue by Just-SKY** [8bitPit/Niagara-Issues#1966]
### Is your feature request related to a problem?

OpenWeather (Extremely poor/useless weather data.)

### Describe the solution you'd like

I know u probably aren't gonna change the weather provider soon. However, I feel Foreca is the best weather provider and more accurate one. I just want to leave at least the idea out there. 

https://corporate.foreca.com/en/foreca-weather-api

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by tjsuominen** [comment]
Or give us an option to choose from a few alternatives?

Not a priority tho as there are hundreds and hundreds of weather apps to choose from for more details.

**Comment by Just-SKY** [comment]
> Not a priority tho as there are hundreds and hundreds of weather apps to choose from for more detail's..
You're right there lots of weather apps out there, but it is sad that such a nice widget is almost useless. Also, it is premium.
However, as I mentioned, it is just a suggestion.

**Comment by Oddward** [comment]
It's premium for the devs as well, for a proper integration providers have a premium paid API, and they'll have to use that specific API in the code to get the data, so throwing in multiple providers isn't trivial or probably very feasible. 

Are you sure this FORECA is more accurate for all/most users? If there's an issue with your location specifically it's advised to install the OpenWeather app through which you can check more accurately and contact those devs to inform them of the issue so they can fix it & improve the UX for all users

**Comment by 8bitPit** [comment]
We're considering switching (back) to Apple Weather (previously DarkSky), but we haven't decided that yet.

**Comment by Just-SKY** [comment]
I'm 100% sure that the majority of the people will appreciate this decision cuz openweather is really bad. Also, I've seen many people talking about this in many channels. 

Thanks for considering this 👊🏼

**Comment by Just-SKY** [comment]
I forgot that I proposed forces as an alternative because of this: 
![A](https://user-images.githubusercontent.com/72170242/187836692-e91e21d3-17fb-4be3-9e03-34f9b114aa63.jpg)
But any other will be better that open weather for sure.

**Comment by 8bitPit** [comment]
No worries, Apple announced their "new" weather API this year, which is basically the API from DarkSky: https://developer.apple.com/documentation/weatherkitrestapi. DarkSkys apps will be shut down, but the Weather app from Apple, for example, uses the same DarkSky API.

**Comment by Just-SKY** [comment]
Mmm. That's great

**Comment by tdpeek3** [comment]
I am one of those people who would prefer to move away from openweather as well. I have to use another weather app for accurate forecasts. I definitely +1 using the DarkSky API!

Thanks for all of your hard work!

**Comment by humzakh** [comment]
Requesting a status update on this issue. Are there still (potential) plans to switch the weather source to Apple WeatherKit?

**Comment by trustosas** [comment]
Bump

**Comment by 8bitPit** [comment]
Yes, we still plan to switch back to DarkSky / WeatherKit. We have even already acquired an API key. Nevertheless, it might still take a while (maybe until 2025?) until we switch providers since we received a lot of other great suggestions we want to implement as well.

**Comment by trustosas** [comment]
It's about time.

**Comment by Shipit215** [comment]
I would love to see the Weather Channel be an option as Google's default Android weather app uses the Weather Channel.  Would be nice if the launcher weather information was the same as Google's default. 
Thanks!
